go/noise/laplace_noise.go

```diff
@@ -203,6 +236,32 @@ func laplaceLambda(l0Sensitivity int64, lInfSensitivity, epsilon float64) float6
 	return l1Sensitivity / epsilon
 }
 
+// getConfidenceIntervalLaplace computes a confidence interval that contains float64 noisedValue with probablility
+// equal to the given confidenceLevel with the given lambda.
+func getConfidenceIntervalLaplace(noisedValue float64, lambda, confidenceLevel float64) ConfidenceInterval {
+	// Finding a symmetrical confidence interval around a Laplace of (0, lambda)
+	// by computing alpha, then calculating Z_(1 - alpha/2) using inverseCDFLaplace.
+	alpha := (1 - confidenceLevel)
+	k := 1 - alpha/2
+	Zk := inverseCDFLaplace(0, lambda, k)
+	// The confidence interval will be noisedValue +/- Z_(1-alpha/2).
+	return ConfidenceInterval{noisedValue - Zk, noisedValue + Zk}
+}
+
+// inverseCDFLaplace returns Z_p where if is Y a random variable then Z_p is the value that satisfies the equation:
+// Pr( Y <= Z_p ) = p where mean and lambda are laplace distribution parameters and p is the probablity.
+func inverseCDFLaplace(mean, lambda, p float64) float64 {
+	return (mean - lambda*sign(p-0.5)*math.Log(1-2*math.Abs(p-0.5)))
```

Comment:
I changed to this but it should be instead:
if p < 1/2
return mean  + lambda * math.Log(2p)
else
return mean - lambda * math.Log(2*(1-p))

because of the already negative sign in the equation.